pkg/activator/handler/handler_test.go

```diff
@@ -82,23 +102,25 @@ func TestActivationHandler(t *testing.T) {
 	act := newStubActivator("real-namespace", "real-name", server)
 
 	examples := []struct {
-		label         string
-		namespace     string
-		name          string
-		wantBody      string
-		wantCode      int
-		wantErr       error
-		attempts      string
-		reporterCalls []reporterCall
+		label           string
+		namespace       string
+		name            string
+		wantBody        string
+		wantCode        int
+		wantErr         error
+		attempts        string
+		endpointsGetter func(activator.RevisionID) (int32, error)
+		reporterCalls   []reporterCall
 	}{
 		{
-			label:     "active endpoint",
-			namespace: "real-namespace",
-			name:      "real-name",
-			wantBody:  "everything good!",
-			wantCode:  http.StatusOK,
-			wantErr:   nil,
-			attempts:  "123",
+			label:           "active endpoint",
```

Comment:
nit: you can dramatically reduce indentation in these cases by using:
```go
{{
   ...
}. {
  ...
}. {
  ...
}}
```

For the array literal.